content/departments/security/tooling/trivy/4-2-0.md

```diff
@@ -1,22 +1,23 @@
+| [CVE-2022-43680](https://github.com/advisories/GHSA-4hjv-8mmr-jxwv)         | sourcegraph/search-indexer, sourcegraph/cadvisor                                                                                                                                                                                                                                             | High         | Info                                                                                                           | Neither Sourcegraph nor its bundled dependencies use the vulnerable functionality reported in this CVE.                                                                                                                                                                                                                                                                                                  |
+| [CVE-2022-42898](https://access.redhat.com/security/cve/cve-2022-42898)     | sourcegraph/minio                                                                                                                                                                                                                                                                            | Medium       | Info                                                                                                           | Neither Sourcegraph nor its bundled dependencies use the vulnerable Kerberos functionality reported in this CVE.                                                                                                                                                                                                                                                                                         | j   |
+| [CVE-2020-7731](https://github.com/advisories/GHSA-gq5r-cc4w-g8xf)          | sourcegraph/grafana, sourcegraph/server                                                                                                                                                                                                                                                      | High         | Low                                                                                                            | This CVE is has been reported in 2020 and it can potentially impact the Sourcegraph instance if SAML2 is used as an authProvider. The impact would be very limited and wouldn't result in disclosure of information or system compromise.                                                                                                                                                                |
```

Comment:
@willdollman do you agree with my reasoning for CVE-2020-7731?

Reply:
How about the following:

> This is a denial of service vulnerability that can affect Sourcegraph instances where SAML2 is configured as an authProvider. The availability impact of exploitation would be limited.